MetadataExtractor/Formats/Exif/makernotes/FujifilmMakernoteDescriptor.cs

```diff
@@ -22,7 +22,8 @@ namespace MetadataExtractor.Formats.Exif.Makernotes
     /// </item>
     /// </list>
     /// </summary>
-    /// <author>Drew Noakes https://drewnoakes.com</author>
    public sealed class FujifilmMakernoteDescriptor : TagDescriptor<FujifilmMakernoteDirectory>
```

Comment:
How did this ever work!?

Reply:
no clue! 🦆

Reply:
Oh the `^M` wasn't visible on the GitHub app. I would have expected this to be normalised.